packages/trpc/server/routers/viewer/bookings/requestReschedule.handler.ts

```diff
@@ -149,24 +147,16 @@ export const requestRescheduleHandler = async ({ ctx, input }: RequestReschedule
   });
 
   // delete scheduled jobs of previous booking
-  const webhookWorkflowPromises = [];
-  webhookWorkflowPromises.push(deleteWebhookScheduledTriggers({ booking: bookingToReschedule }));
+  const webhookPromises = [];
+  webhookPromises.push(deleteWebhookScheduledTriggers({ booking: bookingToReschedule }));
 
-  //cancel workflow reminders of previous booking
-  for (const reminder of bookingToReschedule.workflowReminders) {
-    if (reminder.method === WorkflowMethods.EMAIL) {
-      webhookWorkflowPromises.push(deleteScheduledEmailReminder(reminder.id, reminder.referenceId));
-    } else if (reminder.method === WorkflowMethods.SMS) {
-      webhookWorkflowPromises.push(deleteScheduledSMSReminder(reminder.id, reminder.referenceId));
-    } else if (reminder.method === WorkflowMethods.WHATSAPP) {
-      webhookWorkflowPromises.push(deleteScheduledWhatsappReminder(reminder.id, reminder.referenceId));
-    }
-  }
-
-  await Promise.all(webhookWorkflowPromises).catch((error) => {
-    log.error("Error while scheduling or canceling webhook triggers", JSON.stringify({ error }));
+  await Promise.all(webhookPromises).catch((error) => {
+    log.error("Error while deleting scheduled webhook triggers", JSON.stringify({ error }));
   });
 
+  //cancel workflow reminders of previous booking
+  await deleteAllWorkflowReminders(bookingToReschedule.workflowReminders);
```

Comment:
If we are fine not passing `prisma` as the argument here, then maybe we could simply stop accepting that argument in `deleteAllWorkflowReminders`.